docs/content/guides/3.performance/1.index.md

```diff
@@ -0,0 +1,44 @@
+# Introduction to Web Performance
+::subheader
+Improve your website's user experience and increase conversions by optimizing your website's performance.
+::
+
+Web performance is an ever-important subject in modern web. While your Alokai application is built with the best practices in mind, it's important to understand some of the basics of web performance so that as you build out your custom storefront, you can ensure that your users have a great experience.
+
+On the following pages you will find list of good practices that will help you optimize your website performance.
+
+## Monitoring Performance
+
+### Lighthouse
+
+Lighthouse allows you to test for best SEO, performance, and accessibility practices. These tests run by simulating a user's experience with a specific CPU speed, network speed, and device type. 
+
+These tests run in an isolated environment - so while they can be a way to identify issues, they may not give you a complete picture of how your site performs for real users. 
+
+You can run this Audit using Chrome DevTools or you can even use tools like [Lighthouse CI](https://github.com/GoogleChrome/lighthouse-ci) to run it during your CI/CD pipeline.
+
+In regards to performance, Lighthouse will give you a score for five different metrics:
+
+1. First Contentful Paint (FCP) - the time it takes for the browser to paint the first text or image
+3. Speed Index - how quickly the contents are visible to the user
+4. Total Blocking Time (TBT) - the amount of time after the FCP that the page is blocked from receiving user input
+5. Largest Contentful Paint (LCP) - when the largest contentful element is painted to the viewport
+6. Cumulative Layout Shift (CLS) - the movement of visible elements on the page.
+
+**See More**:
+- [Lighthouse Docs](https://developer.chrome.com/docs/lighthouse/overview)
+- [Lighthouse Metrics](https://developer.chrome.com/docs/lighthouse/performance/first-contentful-paint)
+
+### Google Core Web Vitals
+
+Core Web Vitals uses user data to measure the performance of your website. By aggregating data from real browsers into the [Chrome User Experience Report (CrUX)](https://developer.chrome.com/docs/crux) - you can get a real understanding of how your site is performing for real users.
+
```

Comment:
suggestion: here I would add a note how users can audit their website for CWV and the example of Page Speed Insights https://pagespeed.web.dev/ as we show examples in the Lighthouse section